selendroid-common/src/main/java/io/selendroid/common/SelendroidCapabilities.java

```diff
@@ -395,4 +395,16 @@ private String getDefaultVersion(Set<String> keys, String appName) {
     return listOfApps.size() > 0 ? listOfApps.last() : null;
   }
 
+  // throws exception if user didn't pass the capability as a boolean
+  private Boolean getBooleanCapability(String key) {
+    Object o = getRawCapabilities().get(key);
+    if (o == null) {
+      return null;
+    } else if (o instanceof Boolean) {
+      return (Boolean) o;
+    } else {
+      throw new ClassCastException("DesiredCapability " + key
```

Comment:
I can throw a different exception, i wanted to be more specific than just throwing a RuntimeException but couldn't really think of a better one. I thought about WebDriverException but not really sure if that fits.

The error message will look like this if user passes a string for boolean type of desiredcapability
DesiredCapability emulator's value is expected to be of type Boolean but has been set as type java.lang.String